components/ui/Button.tsx

```diff
@@ -97,7 +97,7 @@ interface ButtonProps {
   title: string;
   disabled?: boolean;
   margin?: Spacing;
-  type?: RNEButtonProps['type'];
+  type?: RNEButtonProps['type'] | 'gradient';
```

Comment:
the `'gradient'` type doesn't exist in the React native elements library, so this change had to be added to the Buttons.tsx